src/dotnet/Microsoft.FxLister/Commands/TypesCommand.cs

```diff
@@ -0,0 +1,78 @@
+using System.CommandLine;
```

Comment:
Any new tools in this repo should go under the "tools" folder and not under the src folder. So please move this took to "tools/fxlister/Microsoft.FxLister".